frontend/src/components/login.js

```diff
@@ -1,44 +1,175 @@
+            setTimeout(() => {
+                setMessage('')
+            }, 1000)
+        }
+        setShowRegister(false);
+    }
+
+    if (showRegister) {
+        return (
+            <div className="col-12 offset-md-4 col-md-4">
+                <form className="card mx-auto px-4 py-2">
+                    <div className="card-body">
+                        <p className="card-title lead text-center mb-4">Register an account</p>
+                        <div className="input mb-3">
+                            <label className="input-text pb-2"
+                                htmlFor="username">username</label>
+                            <input
+                                type="text"
+                                id="rUsername"
+                                name="username"
+                                className="form-control"
+                                aria-label="username-input"
+                                aria-describedby="username-input"
+                                defaultValue={userRegInput}
+                                onChange={(e) => setUserRegInput(e.target.value)} />
+                        </div>
+                        <div className="input mb-3">
+                            <label className="input-text pb-2"
+                                htmlFor="email">email</label>
+                            <input
+                                type="text"
+                                id="rEmail"
+                                name="email"
+                                className="form-control"
+                                aria-label="email-input"
+                                aria-describedby="email-input"
+                                defaultValue={emailRegInput}
+                                onChange={(e) => setEmailRegInput(e.target.value)} />
+                        </div>
+                        <div className="input mb-3">
+                            <label className="input-text pb-2"
+                                htmlFor="password">password</label>
+                            <input
+                                type="password"
+                                id="rPassword"
+                                name="password"
+                                className="form-control"
+                                aria-label="password-input"
+                                aria-describedby="password-input"
+                                defaultValue={passwordRegInput}
+                                style={passwordRegInput === confirmPasswordRegInput && passwordRegInput.length > 1 ? {border: '2px solid green'} : null}
+                                onChange={(e) => setPasswordRegInput(e.target.value)} />
+                        </div>
+                        <div className="input mb-3">
+                            <label className="input-text pb-2"
+                                htmlFor="confirmPassword">confirm</label>
```

Comment:
This label should read 'confirm password'

Reply:
addressed